### PR TITLE
feature/support-sass | Rearrange to support scss styles

### PR DESCRIPTION
**Pull request by zrosenbauer**
Moved js into a `js` dir and added file in `scss` dir. I opened separate PR for the rules that should be reviewed after verifying this one.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,25 +1,26 @@
...
+    "url": "https://github.com/55places/Neighborhoods-JS-Linting.git"
   },
   "bugs": {
     "url": "https://github.com/55places/Neighborhoods-JS-Linting/issues"
   },
-  "main": "index.js",
-  "author": "Jon Rubins <jon.rubins@55places.com>",
+  "main": "js/index.js",
+  "author": "55places.com Team",
   "license": "MIT",
```
Comment (zrosenbauer) [inline package.json]: @jrubins ;)

File: package.json [hunk 2]
```diff
@@ -1,25 +1,26 @@
...
+    "url": "https://github.com/55places/Neighborhoods-JS-Linting.git"
   },
   "bugs": {
     "url": "https://github.com/55places/Neighborhoods-JS-Linting/issues"
   },
-  "main": "index.js",
-  "author": "Jon Rubins <jon.rubins@55places.com>",
+  "main": "js/index.js",
+  "author": "Neighborhoods.com Team",
   "license": "MIT",
```
Comment (zrosenbauer) [inline package.json:19]: @jrubins ;)

Comment (jrubins) [reply package.json:19]: 👎

File: package.json [hunk 3]
```diff
@@ -1,25 +1,26 @@
 {
-  "name": "eslint-config-nhds",
+  "name": "lint-config-nhds",
```
Comment (jrubins) [inline package.json]: This is going to be a breaking change for ESLint: http://eslint.org/docs/developer-guide/shareable-configs (the package needs to be `eslint-config-{name}`. I think we should probably go with sub-packages in this repo.

**Review by jrubins — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (zrosenbauer) [reply package.json]: RTFM but yea that works I'll change that up.

This does raise an issue, if we have two separate packages are we keeping them in sync as far as version? or version each one separately? I liked the idea of the github releases matching up with our package version but we could just version our entire style-guide and each subpackage seperately:

e.g.

``` yaml

github-release: 1.0.0
eslint-version: 2.0.1
sass-lint-version: 1.9.0

```

**Comment by zrosenbauer** [comment]
@jrubins I moved these into two separate packages. We will want to chat on how we are addressing releases.

File: package.json (hunk 2 above)
Comment (cbourdage) [reply package.json:19]: LOL!

**Comment by cbourdage** [comment]
Do we want to discuss this in a FED meeting?

**Review by cbourdage — APPROVED** [review]

**Comment by zrosenbauer** [comment]
@cbourdage @jrubins yea we can chat about that then. I will stay on release 1.0.0 for now.

**Comment by jrubins** [comment]
So I would say that we could model this like Babel. Where the scss "package" has its own `package.json` file and these packages are two different npm dependencies. Thoughts?

**Comment by zrosenbauer** [comment]
@jrubins thats the current setup I'm planning to use...would we then release our Facade Styleguide as a whole everytime we change the packages internally? or not even use releases just version with-in each package.json file?

File: js/package.json [hunk 4]
```diff
@@ -1,25 +1,26 @@
 {
   "name": "eslint-config-nhds",
   "version": "1.0.0",
-  "description": "Neighborhoods.com JavaScript Style Guide",
+  "description": "Neighborhoods.com Facade Style Guide",
```
Comment (jrubins) [inline js/package.json]: Revert? Since we have two packages

File: js/package.json [hunk 5]
```diff
@@ -1,25 +1,26 @@
...
-  "author": "Jon Rubins <jon.rubins@55places.com>",
+  "author": "Neighborhoods.com Team",
   "license": "MIT",
-  "dependencies": {
+  "peerDependencies": {
     "eslint-plugin-import": "2.0.0",
     "eslint-plugin-jsx-a11y": "2.2.2",
-    "eslint-plugin-react": "6.3.0"
+    "eslint-plugin-react": "6.3.0",
+    "sass-lint": "1.9.1"
```
Comment (jrubins) [inline js/package.json]: Remove

File: js/package.json [hunk 6]
```diff
@@ -1,25 +1,26 @@
...
-    "url" : "https://github.com/55places/Neighborhoods-JS-Linting.git"
+    "type": "git",
+    "url": "https://github.com/55places/Neighborhoods-JS-Linting.git"
   },
   "bugs": {
     "url": "https://github.com/55places/Neighborhoods-JS-Linting/issues"
   },
   "main": "index.js",
-  "author": "Jon Rubins <jon.rubins@55places.com>",
+  "author": "Neighborhoods.com Team",
```
Comment (jrubins) [inline js/package.json:18]: Revert? Because it's the right thing to do

File: scss/package.json [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+    "sass",
+    "lint"
+  ],
+  "author": "Neighborhoods.com Team",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/55places/Neighborhoods-JS-Linting/issues"
+  },
+  "homepage": "https://github.com/55places/Neighborhoods-JS-Linting#readme"
+}
```
Comment (jrubins) [inline scss/package.json:26]: Add peer dep of sass-lint here

**Review by jrubins — APPROVED** [review]
A couple of small things, but otherwise looks good.

**Comment by jrubins** [comment]
I'd say we approach versioning like Babel does it. So the first option you said.

File: js/package.json (hunk 6 above)
Comment (zrosenbauer) [reply js/package.json:18]: ;)

File: js/package.json (hunk 4 above)
Comment (zrosenbauer) [reply js/package.json]: +1

File: scss/package.json (hunk 7 above)
Comment (zrosenbauer) [reply scss/package.json:26]: +1

**Pull request merged**